pkg/driver/mysql.go

```diff
@@ -0,0 +1,41 @@
+package driver
+
+import (
+	"context"
+	"database/sql/driver"
+	"github.com/go-sql-driver/mysql"
+	"github.com/pkg/errors"
+)
+
+var errUnknownSysVar = &mysql.MySQLError{Number: 1193}
+
+// setGaleraOpts tries SET SESSION wsrep_sync_wait=7.
+//
+// This ensures causality checks will take place before executing anything,
```

Comment:
Please excuse me being pedantic, but `7` is anything _except_ `SHOW`, which we shouldn't be using here.